### PR TITLE
Minor refinements to terse guide.

### PR DESCRIPTION
**Pull request by LinqLover**
Follow-up from <https://github.com/squeak-smalltalk/squeak.org/pull/44>/complements Help-Squeak-TerseGuide-ct.16.

### CODE REVIEW

File: _pages/terse_guide.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ permalink:  /documentation/terse_guide/
 
 ## Variables
 - Variables must be declared before use
-- Shared vars must begin with uppercase
+- Shared vars (globa,s pools, and class vars) must begin with uppercase
```
Comment (codeZeilen) [inline _pages/terse_guide.md:14]: There seems to be a typo here.

Comment (codeZeilen) [reply _pages/terse_guide.md:14]: @LinqLover Is that a typo or some magic formatting I am not getting? If a typo, I can fix it and merge the pr.

Comment (LinqLover) [reply _pages/terse_guide.md:14]: Oops, sorry :D

```suggestion
- Shared vars (globals, pools, and class vars) must begin with uppercase
```

**Comment by LinqLover** [comment]
Yes, please feel free to merge!

**Comment by LinqLover** [comment]
I just updated the mailing list addresses to HyperKitty in this PR.

**Pull request merged**